item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+
+class Item
+  attr_accessor :genre, :author, :source, :label, :publish_date
```

Comment:
- [ ] Great job so far @sumon766 and @dieum512 You have implemented most of the project requirements. The only thing that I noticed was missing is the custom setter methods that were to be created for the `1-to-many relationships`, allowing items to be added to the collections and establishing the relationships correctly. :100: 
- [ ] So even though using `attr_accessor` does the job, I will suggest you adhere to the project requirements and replace this here with `attr_reader` and create custom setter methods for `genre`, `author`, and `label ` in the `Item class`. `publish_date` is excluded from this. :wink: 
- [ ] Also, since you are just 2 members in this group, kindly remove `source` from the methods since that is only required for teams of 4 :100: 
- Project Requirement
![image](https://github.com/sumon766/my-things/assets/102699958/6a8dd988-4fac-4f55-a125-6be3135bfec7)